app/scripts/modules/ecs/src/serverGroup/configure/wizard/capacityProvider/CapacityProvider.tsx

```diff
@@ -37,10 +65,10 @@ class EcsCapacityProvider extends React.Component<IEcsCapacityProviderProps, IEc
     this.setState({capacityProviderStrategy : capacityProviderStrategy });
   }
 
-  private updateCapacityProviderName = (index: number, targetCapacityProviderName: string) => {
+  private updateCapacityProviderName = (index: number, targetCapacityProviderName: any) => {
```

Comment:
We should explicitly take in the label if that's all we want, instead of taking in any type, define an explicit type of the property(ies) that you need.

Reply:
Correct, the change has been made.